src/__tests__/lib/containers/SynapseTable.test.tsx

```diff
@@ -98,6 +98,14 @@ describe('basic functionality', () => {
     chartSelectionIndex: 0,
     isAllFilterSelectedForFacet: {},
     data: castData,
+    topLevelControlsState : {
```

Comment:
@emmanmills : is there a better way to do this?  mock it using jest perhaps (instead of this real object)?

Reply:
I checked Jest's documentation and it has something called jest.createMockFromModule(moduleName). Is this what you are thinking about? However I am not familiar with the usage.

Reply:
Ok, I'll keep them as stub/real objects for now